### ISSUE TITLE
Docs not available on Docs.rs

### ISSUE THREAD

**Issue by petergarnaes** [zombodb/pgx#9]
The documentation on [docs.rs](https://docs.rs/crate/pgx/0.0.3) are not available due to:
```
docs.rs failed to build pgx-0.0.3
Please check the build logs and, if you believe this is docs.rs' fault, open an issue.
```

**Comment by jamessewell** [comment]
I believe this is to do with `docs.rs` trying to compile three versions of Postgres as part of the build.

A solution is being worked on currently.

**Comment by eeeebbbbrrrr** [comment]
Yeah. We’re trying to find the best way to handle this. 

You can clone this repo and build them locally in the mean time. That works just fine.

**Comment by eeeebbbbrrrr** [comment]
I'm hoping the just-published v0.0.8 will fix this.  Waiting now for docs.rs to pick it up and start a build.

I couldn't get the current docs.rs docker stuff working to test things locally.  :(  I've had success with that in the past, but it seems kinda broken today.  /shrug

**Comment by eeeebbbbrrrr** [comment]
And of course, it still failed b/c I forgot to fix this problem:

`[error] creating /home/crates-build-env/.pgx: Permission denied (os error 13)`

yay me!

Guess we'll wait for the next release...

**Comment by eeeebbbbrrrr** [comment]
just published v0.0.9 to crates.io.  We'll find out soon enough if it'll build on docs.rs.  *crossed fingers*

**Comment by eeeebbbbrrrr** [comment]
This has finally been resolved.  v0.0.10 docs are now on docs.rs, and v0.0.11 should be built within the hour.
